### ISSUE TITLE
Active Killswitch prevents reconnection

### ISSUE THREAD

**Issue by Rafficer** [ProtonVPN/protonvpn-cli-ng#1]
When the killswitch is active (connection dropped) and reconnecting is initiated, the reconnecting won't work:

```
user@debian9:~$ sudo protonvpn r
[!] There was an error connecting to the ProtonMail API.
[!] Please make sure your connection is working properly!
```

Steps to reproduce:

1. Activate Killswitch
2. Connect to any VPN Server
3. Kill the OpenVPN Process via `sudo pkill openvpn`
4. Try to reconnect: `sudo protonvpn r`

**Comment by TauSigma5** [comment]
I made a fix and added a pull request. See: #4

**Comment by Rafficer** [comment]
Fixed through ba22b94
